test/common.js

```diff
@@ -1,7 +1,10 @@
-
```

Comment:
This file isn't, as far as I can tell, used at all any more. It should really go away, I wouldn't extend it. Just put your function directly into `common-tap` above. All the tests should really be loading `common-tap` anyway, as it does some important environment configuration.